src/pages/en/guides/markdown-content.md

```diff
@@ -81,14 +88,14 @@ An example blog post `content` object might look like:
 
 ```json
 {
-  /** Frontmatter from a blog post
+  /** Frontmatter from a blog post **/
   "title": "Astro 0.18 Release",
   "date": "Tuesday, July 27 2021",
   "author": "Matthew Phillips",
   "description": "Astro 0.18 is our biggest release since Astro launch.",
   "draft": false,
   "keywords": ["astro", "release", "announcement"]
-  **/
+  
```

Comment:
Not sure if removing **/ was intentional, but originally this marked the end of the example post's frontmatter props.

Without the ending comment, we don't have any indication of the object properties that came from the Markdown frontmatter and the ones that are injected by Astro ( astro).

```suggestion
  **/
```

Reply:
Yes, I did this intentionally because someone was confused as to why they were commented out.

I was *hoping* that with this comment, and then because after all of these there was a subsequent `astro:` property, it would be obvious what was going on. But, maybe not? 😅

Reply:
> I was hoping that with this comment, and then because after all of these there was a subsequent astro: property, it would be obvious what was going on. But, maybe not? 😅

oh, I totally didn't notice it was actually commenting out the lines 😅 (yup, my mind just forgot that comments, well, comment out code!)

For me it's clear and we don't need to be abundantly clear, but maybe it's better to be safe than sorry?

```suggestion
  /** Injected by Astro **/
```

Reply:
I like Yan’s suggestion here to label the rest of this content.

Reply:
@Yan-Thomas I had to resolve conflics and this got lost. Would you put it back in to the code as it exists now, so it will be easy for me to co-author you?  😅

Reply:
Sure, just a sec.

Reply:
To be honest, @sarah11918, I think we don't need to change anything here anymore. The description of the properties and code sample (now that it is a JavaScript object) sounds a lot more clear to me.

Reply:
🥳